scripts/lib/clean_yarn_cache.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import sys
+
+ZULIP_PATH = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.append(ZULIP_PATH)
+from datetime import date, datetime
+
+from dateutil.relativedelta import relativedelta
```

Comment:
Let's just use `datetime.timedelta(days=180)` -- it's nice for these scripts to not have extra dependencies.